lib/Search/LocalGroups.php

```diff
@@ -26,26 +26,48 @@
 
 namespace OCA\Circles\Search;
 
+use OC;
 use OCA\Circles\ISearch;
 use OCA\Circles\Model\Member;
 use OCA\Circles\Model\SearchResult;
+use OCP\IUser;
+use OCA\Circles\Service\ConfigService;
 
 class LocalGroups implements ISearch {
 
+	/** @var ConfigService */
+	private $configService;
+
+	/**
+	 * @param ConfigService $configService
+	 */
+	public function __construct(ConfigService $configService)
+	{
+		$this->configService = $configService;
+	}
+
 	/**
 	 * {@inheritdoc}
 	 */
 	public function search($search) {
 
 		$result = [];
-		$groupManager = \OC::$server->getGroupManager();
+		$groupManager = OC::$server->getGroupManager();
 
 		$groups = $groupManager->search($search);
+		$user = OC::$server->getUserSession()->getUser();
```

Comment:
should be added as a parameter of the `__construct()`. can you do the same with IGroupManager ?